tests/utils.py

```diff
@@ -0,0 +1,31 @@
+import os
+import pathlib
+from contextlib import contextmanager
+
+import pytest
+
+# TODO mark file skipped in coverage.py
+lhapdf = pytest.importorskip("lhapdf")
+
+test_pdf = pathlib.Path(__file__).parent / "genpdf"
+
+# thanks https://stackoverflow.com/questions/431684/how-do-i-change-the-working-directory-in-python/24176022#24176022
+@contextmanager
+def cd(newdir):
+    prevdir = os.getcwd()
+    os.chdir(os.path.expanduser(newdir))
+    try:
+        yield
+    finally:
+        os.chdir(prevdir)
+
+
+# lets follow the same spirit
+@contextmanager
+def lhapdf_path(newdir):
```

Comment:
I would expose this one in the main package, e.g. for runner use.

Reply:
I agree